esphome/onju-voice-microwakeword.yaml

```diff
@@ -195,9 +195,12 @@ media_player:
             old_volume = new_volume;
 
 micro_wake_word:
-  model: okay_nabu
-  # model: hey_jarvis
-  # model: alexa
+  models:
```

Comment:
How about still using just the model name like the esphome docs show as the first example?
https://esphome.io/components/micro_wake_word.html

Reply:
I think it's fine to be more explicit, with the drawback of lack of "automatic" model updates.

Reply:
Ok your choice - I am guessing esphome automatically selects the v2 folder since the introduction of `models:`

I am giving the name shorthand version and also no specifying a model for `vad:` (like [esphomes example](https://esphome.io/components/micro_wake_word.html#example-usage) gives) a test

Reply:
Year it does not recompiles if I specify a vad model or not so it uses this one by default.
This is now my config:
```
micro_wake_word:
  vad:
  models: 
   #- model: alexa
   - model: okay_nabu
   - model: hey_jarvis
   #- hey_mycroft
```
@jhbruhn `hey_mycroft` is now also available please add it too ^^